packages/php-wasm/universal/src/lib/base-php.ts

```diff
@@ -469,6 +469,33 @@ export abstract class BasePHP implements IsomorphicLocalPHP {
 		);
 	}
 
+	/**
+	 * Prints a message to the console.
+	 * @param message The message to print.
+	 */
+	async helloWorld(message: string) {
+		const phpResponse = await this.run({
+			code: `<?php
+				$fullMessage = "PHP: ${message} \n";
+				echo $fullMessage;
+				`,
+		});
+
+		if (phpResponse.errors) {
+			console.error(phpResponse.errors);
+			return;
+		}
+
+		const cResponse = this[__private__dont__use].ccall(
+			'wasm_hello_world',
+			null,
+			[STRING],
+			[phpResponse.text]
+		);
+
+		console.log(cResponse);
```

Comment:
@adamziel the response I'm getting here is a pointer to a string.
How do I read this string in JS?
I searched around files but couldn't find where we access shared memory.

Reply:
`phpwasm-emscripten-library.js` has some examples:

```js
UTF8ToString(pointer)
```

I'm not sure, though, if that function is exposed outside of the library? You could browse the properties of the Emscripten module (`this[__private__dont__use]`) to see if that method is available there 🤔  If it's not, exposing it could be useful.

Reply:
`this[__private__dont__use].UTF8ToString(cResponse)` worked, thank you!